pkg/kubelet/volume/host_path/host_path.go

```diff
@@ -54,6 +54,12 @@ func (plugin *hostPathPlugin) CanSupport(spec *api.Volume) bool {
 	return false
 }
 
+func (plugin *hostPathPlugin) GetAccessModes() []api.AccessModeType {
+	return []api.AccessModeType{
+		api.ReadWriteOnce,
```

Comment:
Point taken.  I added comments to each volume's GetAccessMode methods to hopefully make it more clear.